kso_utils/widgets.py

```diff
@@ -1249,136 +1220,6 @@ def on_change(change):
 ######################################
 # ###### Tut 5 widgets ###########
 # #####################################
-
-
-def choose_train_params(model_type: str):
```

Comment:
This is a great commit. We should probably ask somebody from Lumi to have a look at it too, though? Just to make sure we did not misunderstand each other :)